spec/nameable_spec.rb

```diff
@@ -0,0 +1,20 @@
+require './nameable'
+require './person'
+
+describe CapitalizeDecorator do
+  person1 = Person.new(26, 'berekethussien')
+  capitalized_name = CapitalizeDecorator.new(person1)
+  trimmed_name = TrimmerDecorator.new(capitalized_name)
+
+  context 'A capitalized_name object' do
+    it 'should return a capitilized name when we call correct_name method' do
+      expect(capitalized_name.correct_name).to eq 'Berekethussien'
+    end
+  end
+
+  context 'A trimmed_name object' do
+    it 'should return a trimmed name when we call correct_name method' do
+      expect(trimmed_name.correct_name).to eq 'Berekethus'
+    end
+  end
+end
```

Comment:
- [ ] Great job on creating test file for `CapitalizeDecorator` and `trimmerDecorator`  class however there are a few classes from decorator classes that are missing the test cases namely, `PersonDecorator` and `Nameable` . Please make sure to include test cases for each of those classes.
- [ ]  Also, please follow the convention of creating separate files for each test case of each class. This will make your tests more organized and easier to maintain. (Also please separate the test case for trimmer and capitalized classes and put them in a separate file of there own)